### PR TITLE
docs(nxdev): update plugin video link

### PR DESCRIPTION
**Pull request by isaacplmann**
Updates embedded video for the plugin documentation

### CODE REVIEW

File: docs/shared/nx-plugin.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Nx plugins are npm packages that contain generators and executors to extend a Nx
 > A list of custom plugins created by the community is found in the [Community](/nx-community) section.
 > Plugins are written using Nx Devkit. **Read [Nx Devkit](/{{framework}}/core-concepts/nx-devkit) for more information.**
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/fC1-4fAZDP4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (FrozenPandaz) [inline docs/shared/nx-plugin.md:9]: Is it just me or is this video private? :eyes:

Comment (isaacplmann) [reply docs/shared/nx-plugin.md:9]: ah, yes

Comment (isaacplmann) [reply docs/shared/nx-plugin.md:9]: try again

Comment (FrozenPandaz) [reply docs/shared/nx-plugin.md:9]: :+1: Yup, it's live now.

**Pull request merged**